ibis/pandas/execution/window.py

```diff
@@ -360,3 +361,65 @@ def execute_series_dense_rank(op, data, **kwargs):
 def execute_series_percent_rank(op, data, **kwargs):
     # TODO(phillipc): Handle ORDER BY
     return data.rank(method='min', ascending=True, pct=True)
+
+
+def ntile(
```

Comment:
this should not be here; see if you can find a better place

Reply:
I added it here because ntile is a window function. could you give more details about why it shouldn't be here? so I can check if I can find a better place for it.

Reply:
ok, make this a private function